portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="contact.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="contact_content">
+    <div id="header">
+        <a href="index.html"> <img src=/images/profile.jpg /> </a>
+        <div class="left">
+            <a href="about.html"> <p> About </p> </a>
+            <a href="achievement.html"> <p> Achievements </p> </a>
+            <a href="#"> <p> Contact </p> </a>
+        </div>
+    </div>
+    <div id="footer">
+        <p> © Andrew Seaman 2020</p> 
+        <div class="left">
+            <a href="https://github.com/andrewseamanco"> <img src="/images/GitHub-Mark-64px.png"> </a>
+            <a href="https://www.facebook.com/andrew.seaman.9210"> <img src="/images/f-logo.png"> </a>
+            <a href="mailto:andrew.seamanco@gmail.com"> <img src="https://pluspng.com/img-png/gmail-vector-png--1600.png"> </a>
+            <a href="https://www.linkedin.com/in/andrew-seaman-42ab53168?trk=people-guest_people_search-card"> <img src="/images/linkedin.png" /> </a>
+        </div>
+    </div>
+    </div>
```

Comment:
Looks like the header and footer divs have the same indentation level as the surrounding contact-content div..  you should indent those inner two one level deeper.

Reply:
Added indent for footer and header